client/scss/components/forms/_nested-panel.scss

```diff
@@ -100,5 +101,7 @@ $guide-line-bottom-margin: calc($form-field-spacing / 3);
   // Nested panels never need the suffix anchor.
   .w-panel__anchor--suffix {
     display: none;
+    width: $header-anchor-size;
+    height: $header-anchor-size;
```

Comment:
Same here. This is a display none override for a specific breakpoint, adding width and height here will do nothing. Please remove.